test/linarith.lean

```diff
@@ -437,6 +438,11 @@ example (p q r s t u v w : ℕ) (h1 : p + u = q + t) (h2 : r + w = s + v) :
   p * r + q * s + (t * w + u * v) = p * s + q * r + (t * v + u * w) :=
 by nlinarith
 
+-- note: much faster than simplex
+example (p q r s t u v w : ℕ) (h1 : p + u = q + t) (h2 : r + w = s + v) :
+  p * r + q * s + (t * w + u * v) = p * s + q * r + (t * v + u * w) :=
+by nlinarith (config := { oracle := some .fourierMotzkin })
```

Comment:
The new algorithm is about 5x slower here than the old one was.

Reply:
It's likely due to the large size of the matrix (about 100 rows and columns), which is actually sparse (only 6% of the entries are non-zero). This is typical of `nlinarith`, and, in the next PR, I'll make it possible to use sparse matrices in the algorithm.

Reply:
Can you edit the comment in this file to explain that simplex is likely slower here due to the use of dense matrices?

Reply:
Done :)